### PR TITLE
net-dns/coredns: Bump to version 1.8.3

### PR DESCRIPTION
**Pull request by zmedico**
* Add fcaps support for CAP_NET_BIND_SERVICE
* Add systemd service unit and tmpfiles config from coredns/deployment repo
* Migrate to GLEP 81 users and groups

Bug: https://bugs.gentoo.org/777429
Package-Manager: Portage-3.0.17, Repoman-3.0.2
Signed-off-by: Zac Medico <zmedico@gentoo.org>

### CODE REVIEW

File: net-dns/coredns/coredns-1.8.3.ebuild [hunk 1]
```diff
@@ -0,0 +1,917 @@
...
+		"TestMetricsAvailableAfterReloadAndFailedReload"
+	)
+	# concat as '|^Test1$|^Test2$|^Test3...$':
+	local known_fail_re="$(printf '|^%s$' "${known_fail[@]}")"
+	# drop '|' in the begining:
+	known_fail_re="${known_fail_re:1}"
+
+	local working_tests_re="$(
+		# get list of all test:
+		{ GOFLLAGS="-mod=readonly" go test -list . ./... ||
```
Comment (thesamesam) [inline net-dns/coredns/coredns-1.8.3.ebuild:899]:
```suggestion
		{ GOFLAGS="-mod=readonly" go test -list . ./... ||
```

File: net-dns/coredns/coredns-1.8.3.ebuild [hunk 2]
```diff
@@ -0,0 +1,917 @@
...
+	"sigs.k8s.io/yaml v1.2.0"
+	"sigs.k8s.io/yaml v1.2.0/go.mod"
+	"sourcegraph.com/sourcegraph/appdash v0.0.0-20190731080439-ebfcffb1b5c0/go.mod"
+)
+
+
+inherit fcaps go-module systemd
+go-module_set_globals
+
+ARCHIVE_URI="github.com/${PN}/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
```
Comment (thesamesam) [inline net-dns/coredns/coredns-1.8.3.ebuild:841]:
```suggestion
ARCHIVE_URI="https://github.com/${PN}/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
```

File: net-dns/coredns/coredns-1.8.3.ebuild [hunk 3]
```diff
@@ -0,0 +1,915 @@
...
+SLOT="0"
+RDEPEND="acct-user/coredns
+	acct-group/coredns"
+
+FILECAPS=(
+	-m 755 'cap_net_bind_service=ei' usr/bin/${PN}
+)
+
+src_compile() {
+	go build -v -ldflags="-X github.com/coredns/coredns/coremain.GitCommit=${EGIT_COMMIT}" || \
```
Comment (juippis) [inline net-dns/coredns/coredns-1.8.3.ebuild:858]: `\` is not needed after `||` for line break.

**Review by juippis — APPROVED** [review]
LGTM, please become the maintainer and drop p-m project with the current maintainer.

**Pull request merged**